### PR TITLE
[docker] remove src directory to reduce size

### PR DESCRIPTION
**Pull request by gjc13**

### CODE REVIEW

File: etc/docker/Dockerfile [hunk 1]
```diff
@@ -39,8 +39,17 @@ WORKDIR /app
...
+RUN rm -rf /app/._build/
 RUN ./script/bootstrap && ./script/setup
 RUN chmod 644 /etc/bind/named.conf.options
+RUN mv /app/script /tmp
+RUN mv /app/etc /tmp
+RUN rm -rf /app/*
+RUN mv /tmp/script /app
+RUN mv /tmp/etc /app
+RUN mkdir -p /app/src/agent
+RUN touch /app/src/agent/main.cpp
```
Comment (bukepo) [inline etc/docker/Dockerfile]: remove this.

File: etc/docker/Dockerfile [hunk 2]
```diff
@@ -39,8 +39,17 @@ WORKDIR /app
 RUN apt-get update
 RUN apt-get install -y git lsb-core sudo
 
+RUN git reset --hard && git clean -xfd
+RUN rm -rf /app/._build/
```
Comment (bukepo) [inline etc/docker/Dockerfile]: will this be cleaned by `git clean -dfx`?

File: etc/docker/Dockerfile [hunk 3]
```diff
@@ -39,8 +39,17 @@ WORKDIR /app
...
 RUN ./script/bootstrap && ./script/setup
 RUN chmod 644 /etc/bind/named.conf.options
+RUN mv /app/script /tmp
```
Comment (bukepo) [inline etc/docker/Dockerfile]: avoid use absolute path? we already set work dir.

File: etc/docker/Dockerfile (hunk 2 above)
Comment (gjc13) [reply etc/docker/Dockerfile]: No, it has to be manually removed.

**Pull request merged**